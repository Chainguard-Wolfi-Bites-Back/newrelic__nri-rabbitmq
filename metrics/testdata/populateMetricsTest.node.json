{
    "partitions": [
        "one",
        "two"
    ],
    "name": "node1",
    "running": true,
    "sockets_used": 523
}