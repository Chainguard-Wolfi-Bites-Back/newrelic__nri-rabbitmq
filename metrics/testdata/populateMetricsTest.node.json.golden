{"event_type":"nodeMetrics","node.fileDescriptorsUsedSockets":523,"node.partitionsSeen":2,"node.running":1}